paasta_tools/setup_chronos_job.py

```diff
@@ -209,6 +210,23 @@ def main():
         log.warn("Skipping %s.%s: Parent job could not be found" % (service, instance))
         sys.exit(0)
 
+    # Check if the service is attempting to access un-whitelisted volumes
```

Comment:
Perhaps we can make create_complete_config take chronos_job_config as an argument (and system_paasta_config while we're at it), to avoid having to reload twice.